helm_deploy/values-preprod.yaml

```diff
@@ -42,7 +42,8 @@ env:
   PATHFINDER_ENDPOINT_API_URL: https://preprod-api.pathfinder.service.justice.gov.uk/
   PIN_PHONES_URL: https://pin-phone-monitor-qa.prison.service.justice.gov.uk/
   REDIS_ENABLED: true
-  SOC_URL: https://manage-soc-cases-preprod.hmpps.service.justice.gov.uk
+  SOC_UI_URL: https://manage-soc-cases-preprod.hmpps.service.justice.gov.uk/
```

Comment:
iirc I think we're trying to have new urls without the trailing slash so that they read nicer - am I correct @JamieDraperUK ?

Reply:
I went with the majority here